Bottom: b232fded7c5166f375b0d8c6731fd0a59d94e850
Top:    497347136291a561f56118b124ad0429541dcdf5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-27 02:16:50 +0100

crosshair.c: Nasty thing FIXME


---

diff --git a/src/crosshair.c b/src/crosshair.c
index 2d4e26d..60318a5 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -693,8 +693,10 @@ CrosshairOn (bool BlockToo)
   if (!Crosshair.On)
     {
       Crosshair.On = true;
+#if 0
       DrawAttached (BlockToo);
       DrawMark (true);
+#endif
     }
 }
 
@@ -707,8 +709,10 @@ CrosshairOff (bool BlockToo)
   if (Crosshair.On)
     {
       Crosshair.On = false;
+#if 0
       DrawAttached (BlockToo);
       DrawMark (true);
+#endif
     }
 }
